source/runbooks/creating-accounts-for-end-users.html.md.erb

```diff
@@ -33,6 +33,12 @@ Customers will fill out the new-environment template via our public repository [
 See an example file [here](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/creating-environments.html#example-json-files).
 
 > Note - If you are adding an environment to an existing application you can add the new environment block to this file, you do not need to create another. Creating multiple environments in bulk is supported, but creating multiple application accounts is not.
+There is also a facility to list only the account which will have approval rights on the accounts. To add this limitation you will need to include the line belo immediately after the "account-type" line.
```

Comment:
Will need a newline above this, otherwise markdown will not introduce a line break.

Reply:
Done